geoportailv3/static/js/layermanager/layermanagerdirective.js

```diff
@@ -45,4 +45,23 @@ app.LayermanagerController.prototype.removeLayer = function(layer) {
 };
 
 
+/**
+ * @param {ol.layer.Layer} layer Layer.
+ * @export
+ */
+app.LayermanagerController.prototype.changeVisibility = function(layer) {
+  var currentOpacity = parseFloat(layer.get('opacity'));
```

Comment:
I'd use `layer.getOpacity` rather than `layer.get('opacity')` to make it clear that "opacity" is a built-in property as opposed to a user-defined property.